src/capi-multiple-paidfor/test.json

```diff
@@ -1,10 +1,10 @@
 {
-    "ComponentTitle": "Barclays Lets Go Forward",
-    "SeriesURL": "barclays-lets-go-forward",
+    "ComponentTitle": "Sweet Veganuary",
+    "SeriesURL": "sweet-veganuary",
     "Brand": "Barclays",
     "TrackingID": "",
```

Comment:
You might want to add a key into the test json too:

```json
  "TrackingPixel": "",
```

Reply:
This is not necessary though! But it may allow you to test adding the pixel 😄

Reply:
This too...